bears/vcs/git/GitCommitBear.py

```diff
@@ -109,6 +113,37 @@ def check_shortlog(self,
                 yield Result(
                     self,
                     "Shortlog of HEAD commit does not match given regex.")
+        if shortlog_imperative_check:
+            colon_pos = shortlog.find(':')
+            shortlog = shortlog[colon_pos:] if colon_pos != -1 else shortlog
+            has_flaws = self.check_imperative(shortlog)
+            if has_flaws:
+                bad_words = ', '.join([words[0] for words in has_flaws])
+                yield Result(self,
+                             "Shortlog of HEAD commit isn't imperative mood,"
+                             "bad words are '{}'".format(bad_words))
+
+    @staticmethod
+    def check_imperative(paragraph):
+        """
+        Check the given sentence/s for Imperatives.
+
+        :param paragraph: The input paragraph to be tested.
+        :returns: A list of invalid words.
```

Comment:
oh please indent the `:returns:` docs to the `paragraph` one. Or use this (new and coming) style:
```
:param paragraph:
    docs...
:returns:
    stuff...
```